Exporter/src/main/java/uk/ac/ebi/biostd/exporter/persistence/dao/SubmissionDao.java

```diff
@@ -94,11 +101,39 @@ public String getUserEmail(long userId) {
                 detailedSubmissionMapper);
     }
 
-    public List<Submission> getStatsSubmissions() {
+    public List<Submission> getSimplifiedSubmissions() {
         return template.query(
-                queries.getSubmissionsStatsQuery(),
-                singletonMap("imagingProjects", statsProperties.getImagingProjects()),
-                statsSubmissionMapper);
+            queries.getSimpleSubmissionsQuery(),
+            singletonMap("imagingProjects", statsProperties.getImagingProjects()),
+            imagingSubmissionMapper);
+    }
+
+    public SubmissionStats getSubmissionStats(Long id) {
+        SubmissionStats stats = new SubmissionStats();
+
+        try {
+            stats = template.queryForObject(
+                queries.getSubmissionsStatsQuery(), singletonMap("submission_id", id), statsSubmissionMapper);
```

Comment:
in general, is not good idea to use exception as a normal case see https://github.com/tatsuyaoiw/effective-java/blob/master/chapter-9.md , here we can query for list and validate if list is empty

Reply:
Nice catch. Done.